docs/ingredients/accounts.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Accounts
+---
+
+# Accounts
+
+#### Takeout
+- Accounts store state (data and lamports)
+- Accounts are used by Programs to store state
+- Accounts are assigned an Owner Program
+- Only an owner program can modify an Accounts state
+- Account owners (i.e Wallets) can sign transactions by providing a signature
+- Signatures are used to verify authenticity, ownership, and approval
+- Verifying the owner of an account and program is important for security
+- Accounts can hold meta data (i.e authority, name, description, price)
+- An authority can be checked on an account to authorize actions inside a program
+- Accounts are charged rent based on space used
+- Accounts can be made rent exempt by supplying a minimum rent balance
+- Accounts are passed into programs using Transactions
+- Programs are actually Accounts marked as executable when deployed
+- Transactions in addition to Instructions and Signatures include Accounts
+- Accounts are marked as read-write, or read-only when including them in transactions
+- Marking accounts as read-only can enable parallel account processing
+- Account owners can provide signatures by signing a transaction
+- Another type of account that can be used by a Program is a PDA
+- A PDA (Program derived Account) gives a program a way provide a signature
+
+#### Delivery
+Accounts store state (data and lamports). Lamports are fractional units of native SOL (0.000000001). While programs themselves do not store state directly, accounts can be used to store all the state a program requires. How does this work? Accounts when setup are assigned an owner. The owner is a program that has exclusive control over any changes to state. The owner program is referenced using a program id (address, or publickey). Anyone can read an account, but only the program that owns the account has the authority to change or permit modifications to the account data, including any debits or transfers from an account balance. If an account is able to recieve SOL or Tokens, anyone can transfer or send a balance to the account. If an account is not owned by a program, the program is only permitted to read its data and credit the account. Why might an account not be able to recieve SOL? 
```

Comment:
Isn't lamports considered a state of an account owned by the systemProgram? Is that info needed?

It may be good to explain the two different types of accounts, and go from there. That'll help with the explanations of sending/receiving SOL/SPL-Tokens and so on.